From dfa3818f38d12419082f382a8c3e76b52250e336 Mon Sep 17 00:00:00 2001
From: Dave Lin <dave.lin@intel.com>
Date: Wed, 20 Apr 2016 16:09:53 +0800
Subject: [PATCH 1/1] EM/PMIC_CCSM: Fixup of notify OTG for all PC ports source
 connect/disconnect

This patch fixes up the missing changes of the following patch during
synchronization from cht-common-3.14.55 quilt:
EM-PMIC_CCSM-Notify-OTG-for-all-PC-ports-source-conn.patch

Change-Id: I25aa2f15524a4d1d36fe76d41d3191dbfb38a562
Signed-off-by: Dave Lin <dave.lin@intel.com>
---
 drivers/power/intel_pmic_ccsm.c | 8 +-------
 1 file changed, 1 insertion(+), 7 deletions(-)

diff --git a/drivers/power/intel_pmic_ccsm.c b/drivers/power/intel_pmic_ccsm.c
index fad76c8..52453cf 100644
--- a/drivers/power/intel_pmic_ccsm.c
+++ b/drivers/power/intel_pmic_ccsm.c
@@ -955,13 +955,7 @@ static void handle_internal_usbphy_notifications(int mask)
 	case POWER_SUPPLY_CHARGER_TYPE_ACA_C:
 	case POWER_SUPPLY_CHARGER_TYPE_MHL:
 	case POWER_SUPPLY_CHARGER_TYPE_B_DEVICE:
-		/* set inlimit to 1500 only if
-		 * platform is USB compliant
-		 */
-		if (chc.pdata->usb_compliance)
-			cap.ma = ACINPUTICC1500VAL;
-		else
-			cap.ma = HIGH_POWER_CHRG_CURRENT;
+		cap.ma = HIGH_POWER_CHRG_CURRENT;
 		/*
 		 * don't notify to OTG driver incase of non PC/Dock ports
 		 * connect/disconenct.
-- 
1.9.1

